orchestra/static/orchestra/dashboard/tasktable.scss

```diff
@@ -1,11 +1,3 @@
 .collapsed-toggle {
   cursor: pointer;
 }
```

Comment:
based on the comment below, after figuring out where to edit scss, can you QA to make sure that
- tags still show up on the Dashboard
- tags now show up in Available Tasks?

Reply:
After the change, they still show up!!